### PR TITLE
Modify the Best Practices for DeFi Project Section

### PR DESCRIPTION
**Pull request by aw126**
Enrich the contents for Best Practices for DeFi Project on Cronos.

### CODE REVIEW

**Review by zkaizhi — COMMENTED** [review]
Is it possible to immediately update this in gitbook?

File: docs/getting-started/defi-practice.md [hunk 1]
```diff
@@ -3,25 +3,70 @@
 
 Decentralized Finance (DeFi) is revolutionizing the finance industry and bringing boundless opportunities to the finance world. However, at the same time it has witnessed millions of funds being hacked. We believe it is critical for us to play a part to minimize such incidents as we place security as a top priority. With that, we have compiled a list of best practices to develop secure smart contracts for the Cronos ecosystem. This is by no means an exhaustive list but we believe it is a good foundation and minimum security baseline for any smart contract project.
 
-## Secure Coding Practices:
+## Secure Coding Practices
 
+### Design
+#### Documentation
```
Comment (zkaizhi) [inline docs/getting-started/defi-practice.md:9]: Could we use at max 3 levels, #, ## and ### because 4 don't work on gitbook

**Review by zkaizhi — APPROVED** [review]

File: docs/getting-started/defi-practice.md (hunk 1 above)
Comment (aw126) [reply docs/getting-started/defi-practice.md:9]: Sure, thanks for brining it up.

**Pull request merged**